src/components/Questionnaire/QuestionnaireResults/Text/ResultDescriptionILS.tsx

```diff
@@ -42,13 +47,15 @@ const ResultDescriptionILS = ({ ILSdim = ILSDimension }: ResultDescriptionILSPro
       balancedDimensionsArray.push(dimensionArray[index] + '.' + interpretationArray[index])
     } else {
       unbalancedDimensionsArray.push(
-        <div key={'Dimension: ' + dimensionArray[index] + ' Interpretation: ' + interpretationArray[index]}>
+        <div
+          data-testid={'Dimension: ' + dimensionArray[index] + ' Interpretation: ' + interpretationArray[index]}
```

Comment:
Warum wird hier in ein array gepusht 😂 ich rieche ein disrespect gegenüber functional programming xd

Reply:
Bro der code ist schon so alt wie meine Tochter, ich möchte den nicht mehr anfassen 😭

Reply:
Halt wirklicj so alt hahahaha. Ja wir machen issue auf dafür. Für wann anders  :)

Reply:
Hab issue erstellt: [Issue](https://github.com/HASKI-RAK/HASKI-Frontend/issues/212)